multimodal/src/autogluon/multimodal/models/mmocr_text_detection.py

```diff
@@ -66,22 +66,23 @@ def __init__(
 
         # TODO: the logic here (line69 ~ line75) could be shared across multiple mmlab code, consider wrap them in utils.py.
         # download config and checkpoint files using openmim
-        checkpoints = download(package="mmocr", configs=[checkpoint_name], dest_root=".")
+        # checkpoints = download(package="mmocr", configs=[checkpoint_name], dest_root=".")
 
-        # read config files
-        assert mmcv is not None, "Please install mmcv-full by: mim install mmcv-full."
-        config_file = checkpoint_name + ".py"
-        if isinstance(config_file, str):
-            self.config = mmcv.Config.fromfile(config_file)
+        # # read config files
+        # assert mmcv is not None, "Please install mmcv-full by: mim install mmcv-full."
+        # config_file = checkpoint_name + ".py"
+        # if isinstance(config_file, str):
+        #     self.config = mmcv.Config.fromfile(config_file)
 
-        # build model and load pretrained weights
-        assert mmocr is not None, "Please install MMOCR by: pip install mmocr."
+        # # build model and load pretrained weights
+        # assert mmocr is not None, "Please install MMOCR by: pip install mmocr."
 
-        checkpoint = checkpoints[0]
-        self.model = build_detector(self.config.model, test_cfg=self.config.get("test_cfg"))
-        if checkpoint is not None:
-            checkpoint = load_checkpoint(self.model, checkpoint, map_location="cpu")
+        # checkpoint = checkpoints[0]
+        # self.model = build_detector(self.config.model, test_cfg=self.config.get("test_cfg"))
+        # if checkpoint is not None:
+        #     checkpoint = load_checkpoint(self.model, checkpoint, map_location="cpu")
```

Comment:
Consider cleaning up the commented code.

Reply:
Done.